test/e2e/pkg_test.go

```diff
@@ -32,140 +32,89 @@ import (
 // Providers, Configurations, etc).
 const LabelAreaPkg = "pkg"
 
-func TestConfiguration(t *testing.T) {
-	// Test that we can install a Configuration from a private repository using
-	// a package pull secret.
+// TestConfigurationPullFromPrivateRegistry tests that a Configuration can be
+// installed from a private registry using a package pull secret.
+func TestConfigurationPullFromPrivateRegistry(t *testing.T) {
```

Comment:
@negz, both this and the test below have a `configuration/crossplane.yaml` file defined, which looks like wasn't used even on master, can we delete them or am I missing something? I wonder if we could think of something to enforce that all fixtures are actually used to avoid having them drift away unnoticed

Reply:
Obviously immediately writing this comment I understood the reason of those files 👍 They were actually used to build the Configuration used in the test 👍